bundles/pom.xml

```diff
@@ -156,6 +156,7 @@
     <module>org.openhab.binding.plugwise</module>
     <module>org.openhab.binding.powermax</module>
     <module>org.openhab.binding.pulseaudio</module>
+    <module>org.openhab.binding.pushbullet</module>
```

Comment:
Could you add the binding to the codeowners

Reply:
The first file in the "changes" tab shows the addition to CODEOWNERS.

I'll wait for your merge, and if does not show up in the HEAD , I'll create an additional PR as suggested